### PR TITLE
atx.intro1.escalante.adriana

### PR DESCRIPTION
**Pull request by adriescalante**

### CODE REVIEW

**Comment by e-zambrano** [comment]
@adriescalante - excellent work on adding headers elements. Great work

**Comment by e-zambrano** [comment]
@adriescalante - Everything looks good. Keep up the good work!

**Comment by e-zambrano** [comment]
@adriescalante - All good over here. Just make sure you beautify your HTML and CSS before sending it. Beautify was the package we installed on Atom to format the code in a nice way

File: 11LessonEleven/in-class-project/css/style.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+header{
+  background-color: #33A8DD;
+  color: white;
+  width: 100%;
+  height:250px;
+  text-align: left;
+}
+
+.clearfix{
```
Comment (e-zambrano) [inline 11LessonEleven/in-class-project/css/style.css:89]: I would rather you leave the `.clearfix` class just for the purpose of fixing parent collapse and use another selector for that element

File: 08LessonEight/homework/exercise-06.html [hunk 2]
```diff
@@ -1,10 +1,38 @@
...
+
+<head>
+    <title>Exercise 06</title>
+    <link rel="stylesheet" href="css/exercise-06-style.css">
+</head>
+
+<body>
+    <section>
+        <article class="truck">
+            <img src="img/truck.jpg" />
```
Comment (e-zambrano) [inline 08LessonEight/homework/exercise-06.html:12]: For some reason the image does not display. try changing `jpg` for `JPG`

File: 13LessonThirteen/in-class-project/index.html [hunk 3]
```diff
@@ -1,10 +1,82 @@
...
+                    </p>
+                    <p>
+                        Twitter
+                    </p>
+                </section>
+            </section>
+        </footer>
+    </main>
 
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
```
Comment (e-zambrano) [inline 13LessonThirteen/in-class-project/index.html:79]: this script get you jQuery, you are missing the one that gets you the JavaScript library for Bootstrap. Check the template I sent to the channel

File: 13LessonThirteen/homework/css/style.css [hunk 4]
```diff
@@ -17,16 +17,18 @@ font-size: 60px;
 font-weight: 500;
 }
 
-.keyboard .geo .building{
+.keyboard .geo .building .landscape{
```
Comment (e-zambrano) [inline 13LessonThirteen/homework/css/style.css:20]: remember, avoid `descendant selector` with more than three `descendants`

File: 13LessonThirteen/homework/css/style.css [hunk 5]
```diff
@@ -17,16 +17,18 @@ font-size: 60px;
...
   height:198px;
   padding-top: 10px;
 }
-
-.landscape img{
+.top-left {
+      height: 297px;
+}
+/*.landscape img{
```
Comment (e-zambrano) [inline 13LessonThirteen/homework/css/style.css:27]: if something is commented better just remove it

**Comment by e-zambrano** [comment]
@adriescalante - great job. just remember to `beautify` your code before committing

File: 09LessonNine/in-class-project/index.html [hunk 6]
```diff
@@ -1,10 +1,17 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Intel</title>
+  <title>Gardening & Landscaping</title>
```
Comment (zacacollier) [inline 09LessonNine/in-class-project/index.html:4]: remember to use `&amp;` for the `&` symbol in text elements

File: 09LessonNine/in-class-project/index.html [hunk 7]
```diff
@@ -1,10 +1,17 @@
...
+  <link rel="stylesheet" href="css/css-reset.css"/>
   <link rel="stylesheet" href="css/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto+Condensed" rel="stylesheet">
 </head>
 <body>
-
+  <header>
+    <h4>Landscaping and More</h4>
+    <h1>We Craft Your Garden</h1>
+    <h3>check our gardening services and tools</h3>
```
Comment (zacacollier) [inline 09LessonNine/in-class-project/index.html:13]: Don't forget to keep your fonts consistent - this `h3` is rendering differently from the `h1` and `h4` above

File: 09LessonNine/in-class-project/css/style.css [hunk 8]
```diff
@@ -0,0 +1,142 @@
...
+  background-color: #6e6e6e;
+  background-size: cover;
+  width: 69%;
+  float: right;
+  clear: right;
+  height: 530px;
+}
+
+.growing{
+  background-image: url("../img/featured-image-3.jpg");
```
Comment (zacacollier) [inline 09LessonNine/in-class-project/css/style.css:73]: hmm can you think of why your `background image` isn't filling its container element?

File: 09LessonNine/in-class-project/css/style.css [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+padding-right: 10px;
+}
+
+a{
+  text-decoration: none;
+  font-family:'Roboto Condensed', sans-serif;
+  font-size: 10px;
+}
+
+/*CSS for service.html starts here*/
```
Comment (zacacollier) [inline 09LessonNine/in-class-project/css/style.css:53]: Good practice! Comment liberally to keep your code well-organized

File: 14LessonFourteen/in-class-project/css/style.css [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+  padding: 20px;
+}
+
+.geo .building{
+  clear:right;
+}
+
+
+
+@media (max-width: 768px){
```
Comment (zacacollier) [inline 14LessonFourteen/in-class-project/css/style.css:43]: Great work with these media queries!

**Pull request closed without merging**